apps/verticalhome/style/css/app.css

```diff
@@ -104,3 +99,38 @@ body.edit-group {
   font-weight: 300;
 }
 
+#blinknotif{
+  position: fixed;
+  left: 0px;
+  bottom: 3.2rem;
+  right: 0px;
+  background-color: #555;
+  color: #FFF;
+  z-index: 99;
+  content: "";
+  clear: both;
+  display: block;
```

Comment:
I think this display-property is unnecessary. It is overridden below at line 117